### ISSUE TITLE
Couple of errors when running demo tests

### ISSUE THREAD

**Issue by HelloWorldLTY** [GSK-AI/seq2cells#1]
Hi, thanks for your great work! I am trying to run the test codes to understand the training process, however, I faced many problems here.

First, I cannot run the preprocess step:
```
sh repro_preprocess.sh <hg38.fa>
```
The error is: bash: syntax error near unexpected token `newline'.

If I delete the argument <hg38.fa>, then the error becomes:

  File "/gpfs/gibbs/project/zhao/tl688/conda_envs/seq2cells/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/gpfs/gibbs/project/zhao/tl688/conda_envs/seq2cells/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/gpfs/gibbs/project/zhao/tl688/conda_envs/seq2cells/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/gpfs/gibbs/project/zhao/tl688/conda_envs/seq2cells/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1735, in _make_engine
    self.handles = get_handle(
  File "/gpfs/gibbs/project/zhao/tl688/conda_envs/seq2cells/lib/python3.9/site-packages/pandas/io/common.py", line 856, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'toy_query_tss_out.tsv'

If I try to jump this step and run the training code directly, the error becomes:

Traceback (most recent call last):
  File "/gpfs/gibbs/pi/zhao/tl688/seq2cells/tests/../seq2cells/scripts//training/fine_tune_on_anndata.py", line 44, in <module>
    from seq2cells.data.enformer_tss_based import AnnDataEmbeddingTssDataset
ModuleNotFoundError: No module named 'seq2cells'

And I think the running code should be 

```
sh repro_train.sh
```

not 

```
sh repro_train_sh
```

Could you please help me? Thanks a lot.

**Comment by HelloWorldLTY** [comment]
I have modified the /path/to/dict to the currect path, however, the bug still exists. I will be great if you can upload a new version. Thanks.

**Comment by rschwess** [comment]
Hi HelloWorldLTY, 

Thanks for your interest.


1. For the first issue:

```
sh repro_preprocess.sh <hg38.fa>
```

Have you pointed it to an existing reference genome file on your system? Or have you downloaded one as detailed in the notes at the end of the **Test** paragraph? 

`wget https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz`

Then run as 

```
sh repro_preprocess.sh ./hg38.fa.gz
```


2. For the second issue:

Can I confirm that you have followed the install isntructions under **Install** and specifically that you add seq2cells to your PYTHONPATH?

e.g. 

`export PYTHONPATH="${PYTHONPATH}:/Path/to/location"`

Where `/Path/to/location` refers to where you saved seq2cells locally.

**Comment by HelloWorldLTY** [comment]
Hi, thanks a lot. now I have addressed this issue by setting the correct path of seq2cells. Moreover, it is good to know I should use .gz file of hg38 rather than the unzipped version.

Moreover, I tried to run the sequence mode in the prediction step, but it told me that the sequence mode has not been implemented. Does this mean in the future you plan to release methods for the prediction based on sequence data directly? Thanks.

**Comment by rschwess** [comment]
Side note: It works both with a gzipped and unzipped reference genome.

I don't follow where exatly you tried running in sequence mode. That should be in the inetuning script? 

For finetuning we only implemented the workflow from precomputed embeddings yet. I'm afraid we have no short term plan to implement the sequence mode as we found the workflow via precomputed embeddings much more useful in practice for single cell data.

**Comment by HelloWorldLTY** [comment]
It makes sense to me. Thanks a lot!

**Comment by rschwess** [comment]
Side note: It works both with a gzipped and unzipped reference genome.

I don't follow where exatly you tried running in sequence mode. That should be in the inetuning script? 

For finetuning we only implemented the workflow from precomputed embeddings yet. I'm afraid we have no short term plan to implement the sequence mode as we found the workflow via precomputed embeddings much more useful in practice for single cell data.
